src/Sulu/Bundle/ContentBundle/Resources/public/js/components/internal-links/main.js

```diff
@@ -236,7 +237,9 @@ define([], function() {
             return templates.contentItem(
                 item[this.options.idKey],
                 item.title,
-                this.getItemIcons(item)
+                item.url,
+                this.getItemIcons(item),
+                this.sandbox.util.cropMiddle
```

Comment:
Ah, different lengths, got it... Maybe we should introduce the check then. Same for the content I put below.